runtime/interpreter/function.go

```diff
@@ -170,8 +170,10 @@ func (f HostFunctionValue) Invoke(invocation Invocation) Value {
 }
 
 func (f HostFunctionValue) GetMember(_ *Interpreter, _ func() LocationRange, name string) Value {
-	value, _ := f.Members.Get(name)
-	return value
+	if variable, ok := f.NestedVariables.Get(name); ok {
```

Comment:
should we check whether `f.NestedVarable` is nil, since [NewHostFunctionValue](https://github.com/onflow/cadence/pull/715/files#diff-bd8f574d953b36c96ac20b3a19aee38b0189d4711eb452622caabb6465bcfb24L122) constructor-alike function only acecpts the `HostFunction`.

Reply:
Good point! Fixed in 501f6563e6fa825e40a6296e24f8a4a50bd939a7